### ISSUE TITLE
Start up using pins ( from another pi)

### ISSUE THREAD

**Issue by bortek** [PiSupply/Pi-Supply-Switch#13]
I understand that this can be used to cut the power off but is there a way to start up pi using Pins and not a button? My intention is to use another pay to control "main" pi both cutting power and starting it up (cold reset).

**Comment by tvoverbeek** [comment]
Since SW1 can be used to both shut down (and power off) the Pi (long press) and start it up (short press) you should be able to use the unpopulated header J5 which is in parallel to SW1. Provide a short 3.3V pulse to start it up and a long pulse for shutting it down. For timing see the SW1 settings in the GUI/CLI. Check which pin of J5 has the constant 3.3V and connect the other one to an GPIO output of the other Pi.

**Comment by bortek** [comment]
Hmm that should probably work. Let me get the board first and test it. 

Do I need any resistors between GPIO and pins or can I connect them directly ?

**Comment by tvoverbeek** [comment]
The PiJuice already has a 3k9 resistor between the J5 pin and the sw input.
Hence you should be able to connect the GPIO output from the other pi to the J5 pin directly.

**Comment by bortek** [comment]
I was planning to use PiSwitch only, without using PiJuice. Is it possible? And if yes then how do I do about 3.9 resistor?

**Comment by tvoverbeek** [comment]
@bortek That is something different from what you wrote originally.
I understood that you wanted to use the main Pi + Pijuice combination controlled by a second RPi.
Do you mean main Pi + PiSwitch controlled from a second RPi?
I do not know the PiSwitch details, but after a quick look at the schematic and docs I do not think its possible.

**Comment by bortek** [comment]
Sorry for making a confusion. Let me rephrase my question. Is it possible to control Pi-Supply-Switch connected to RPi from a second RPi using for example GPIOs to signal Pi-Supply-Switch to turn off and on?
